metasrv/src/watcher/watcher_manager.rs

```diff
@@ -209,10 +205,8 @@ impl WatcherManagerCore {
             filter,
         });
 
-        let _ = watcher_stream.send(WatchResponse {
-            watch_id: watcher_id,
-            event: None,
-        });
+        // send None event to notify stream has been created.
+        let _ = watcher_stream.send(WatchResponse { event: None });
```

Comment:
Since create/cancel is removed, the client may not need to be notified anymore?

Reply:
maybe notified at the first time the watcher been created?

Reply:
I don't know.

Usually, when I create a watcher, I'll expect no message to return until something I'm interested in happens.